diff --git a/src/log.cpp b/src/log.cpp
index 963005a4..1c8af118 100644
--- a/src/log.cpp
+++ b/src/log.cpp
@@ -100,7 +100,7 @@ static void removeLinesFromFile(int64_t sizeToSkip) {
 		boost::filesystem::rename(tempFilePath, logFile.path);
 	}
 	catch (const boost::filesystem::filesystem_error &) {
-		boost::filesystem::copy_file(tempFilePath, logFile.path, boost::filesystem::copy_option::overwrite_if_exists);
+		boost::filesystem::copy_file(tempFilePath, logFile.path, boost::filesystem::copy_options::overwrite_existing);
 		boost::filesystem::remove(tempFilePath);
 	}
 
